libtbx/auto_build/bootstrap.py

```diff
@@ -775,7 +775,7 @@ class geostd_module(SourceModule):
 
 class boost_module(SourceModule):
   module = 'boost'
-  anonymous = ['git',
+  anonymous = ['git', '-b 1.70',
```

Comment:
Would suggest updating the repository on merge, rather than explicitly pointing to a branch

Reply:
That works, but once this pull request is approved, we should decide on a date to merge and to advertise that date so that developers are not caught off guard about this change. People will probably have to rebuild completely or delete their build/lib/*boost* files.